app/app/explore/components/ExploreTable.tsx

```diff
@@ -0,0 +1,359 @@
+import { useState, useMemo, useEffect } from "react";
+import Image from "next/image";
+import TranslucentCard from "@/app/TranslucentCard";
+import TopCards from "@/app/explore/TopCards";
+import { DoubleArrow } from "@shared/components/svg";
+import { PoolDetail, PoolDetails } from "@/midgard";
+import {
+  addDollarSignAndSuffix,
+  calculatePoolTVL,
+  calculateVolumeDepthRatio,
+  calculateVolumeUSD,
+  calculateSaverTVL,
+  formatNumber,
+  getFormattedPoolTVL,
+  getFormattedSaverTVL,
+} from "@/app/utils";
+import SortHeader, { SortKey } from "./SortHeader";
+
+interface Saver {
+  asset: string;
+  saversCount: number;
+  saversReturn: string;
+  earned: string;
+  filled: number;
+  assetPriceUSD: string;
+  saversDepth: string;
+  assetDepth: string;
+  synthSupply: string;
+}
+
+enum SortDirection {
+  ASC = "asc",
+  DESC = "desc",
+}
+
+interface SortConfig {
+  key: SortKey;
+  direction: SortDirection;
+}
+
+interface Column {
+  key: string;
+  label: string;
+  width: string;
+  sortable?: boolean;
+  render: (item: any) => React.ReactNode;
```

Comment:
We should also be specifying concrete types. Using `any` isn't a good practice since it will obfuscate runtime errors that should have been caught by the compiler.